test/parser/test_type_currency.py

```diff
@@ -25,8 +25,17 @@ class TestModel(BaseModel):
         ("$1,234.5678", 1234.5678),
         ("   $1,234  ", 1234.0),
         ("   $ 1,234  ", 1234.0),
+        ("1.234 €", 1234.0),
+        ("1.234€", 1234.0),
+        ("€1.234", 1234.0),
         ("1,234$", 1234.0),
         ("1,234 $", 1234.0),
+        ("1.234,45", 1234.45),
+        ("1.234.456,00", 1234456.00),
+        ("1.000.000", 1000000.00),
+        ("Starting at: 12:00", 12.0),
+        ("Starting At 12.99", 12.99),
+        ("From 399.99", 399.99),
```

Comment:
can we add a fraction case eg: 0.0004 $
this is valid for stuff like crypto or stocks

Reply:
done!